hnn_core/parallel_backends.py

```diff
@@ -57,32 +61,45 @@ def _gather_trial_data(sim_data, net, n_trials, postproc):
     """
     dpls = []
 
+    # Create array of equally sampled time points for simulating currents
+    cell_type_names = list(net.cell_types.keys())
+    cell_response = CellResponse(times=sim_data[0]['times'],
+                                 cell_type_names=cell_type_names)
+    net.cell_response = cell_response
+
     for idx in range(n_trials):
-        dpls.append(sim_data[idx][0])
-        sim_data_trial = sim_data[idx][1]
-        net.cell_response._spike_times.append(sim_data_trial[0])
-        net.cell_response._spike_gids.append(sim_data_trial[1])
+
+        # cell response
+        net.cell_response._spike_times.append(sim_data[idx]['spike_times'])
+        net.cell_response._spike_gids.append(sim_data[idx]['spike_gids'])
         net.cell_response.update_types(net.gid_ranges)
-        net.cell_response._vsoma.append(sim_data_trial[3])
-        net.cell_response._isoma.append(sim_data_trial[4])
+        net.cell_response._vsoma.append(sim_data[idx]['vsoma'])
+        net.cell_response._isoma.append(sim_data[idx]['isoma'])
+
+        # extracellular array
         for arr_name, arr in net.rec_arrays.items():
```

Comment:
```suggestion
        for arr_name in net.rec_arrays.keys():
```

The `arr` isn't used!

Reply:
I decided to use `arr` to make the following lines a bit more readable